emoji_list.py

```diff
@@ -72,24 +67,19 @@
     "fearful": "\xf0\x9f\x98\xa8",
     "cold_sweat": "\xf0\x9f\x98\xb0",
     "disappointed_relieved": "\xf0\x9f\x98\xa5",
-    "sweat": "\xf0\x9f\x98\x93",
     "hugging": "\xf0\x9f\xa4\x97",
     "hugging_face": "\xf0\x9f\xa4\x97",
     "thinking": "\xf0\x9f\xa4\x94",
     "thinking_face": "\xf0\x9f\xa4\x94",
     "face_with_hand_over_mouth": "\xf0\x9f\xa4\xad",
-    "yawning_face": "\xf0\x9f\xa5\xb1",
     "shushing_face": "\xf0\x9f\xa4\xab",
     "lying_face": "\xf0\x9f\xa4\xa5",
     "liar": "\xf0\x9f\xa4\xa5",
     "no_mouth": "\xf0\x9f\x98\xb6",
```

Comment:
I think we should also remove `:no_mouth:` 
![image](https://user-images.githubusercontent.com/6956898/149633930-793c1caa-7e11-4ead-af50-48edbce0e60f.png)

```suggestion
```
The throwing up one is pretty funny, so I say we keep that one. Other than that, looks good!